Makefile

```diff
@@ -50,16 +46,15 @@ clean-all: clean clean-node clean-txs
 test-legacy: ## Runs test suite with output from tests printed
 	@cargo test -- --nocapture -Zunstable-options --report-time
```

Comment:
No need for `test-legacy` target since we can now use `--no-capture` with nextest